paper-ieee-trans.tex

```diff
@@ -63,61 +63,63 @@
 applications, has been driven by the needs of scientific communities as part of their
-pan-European research collaborations. One of the well-known examples is the Worldwide 
-Large Hadron Collider (LHC) Computing Grid infrastructure, powered 
-by the particle physics scientific community, built up to satisfy the computing needs
-of processing, storing and analysing the huge quantity of data coming from the 
+pan-European research collaborations. One such example is the Worldwide 
+Large Hadron Collider (LHC) Computing Grid infrastructure, motivated 
+by the particle physics scientific community, build to satisfy the computing needs
+of processing, storing and analysis of the very large amount of data produced by the
 LHC particle accelerator. A significant number of innovation projects and
-technology transfer activities -- embraced by the 5th, 6th and 7th Framework Programmes,
-and ultimately by the Horizon 2020 programme \cite{h2020} -- are being held to coordinate and 
-develop distributed solutions that provide a seamless and transparent access to those
-computing and data resources, available for the researchers.
-
-Major developments were done throughout the years, mainly focused on providing new
-functionalities, leaving aside in the early times the adoption of software engineering methodologies.
-The software delivered was then less tested, resulting in constant
+technology transfer activities -- from the 5th, 6th and 7th Framework Programmes,
+and recently by the Horizon 2020 programme \cite{h2020} -- are under way to
+develop software solutions that provide a seamless and transparent access to distributed
+computing and data resources by the researchers.
+
+Major developments were achieved throughout the years focused on providing new
+functionalities, partially neglecting the adoption of software engineering methodologies, in the early days.
+The released software was less tested, resulting in a high rate of
 rollbacks and patches applied to production systems. Thereby, the stability and
 reliability of the e-Infrastructures was often compromised, affecting
 negatively their exploitation by the scientific communities.
 
-Soon it become apparent that a balance was needed between new features addition
-and the reliability of the software produced. As a consequence, a substantial part of the
-software lifecycle would have to be devoted to deal with the quality of software being
-delivered. At this point, evolving software engineering practices were
-considered and gradually adopted to define the quality procedures, teams
-organization and pilot testbeds.
+Soon it become apparent the need for a better balance between the addition of new features
+and the reliability of the released software. Consequently, a substantial part of the
+software lifecycle would have to be devoted to the quality of the software being
+delivered. The evolution of software engineering practices were
```

Comment:
it seems to me that with this sentence we are considering the "practices" (which are evolving ones) not the "evolution"